jest-puppeteer.config.js

```diff
@@ -16,6 +16,8 @@ module.exports = {
       height: 400,
     },
     args: [
+      // so that tests can be run as root, if you're into that sort of thing.
+      '--no-sandbox',
```

Comment:
Could this be something like `process.env.CI === 'true' ? '--no-sanbox' : undefined` ? That way puppeteer doesn't reuse cookies when running tests locally?

Reply:
yea definitely, i'll change